conference/hosting-guidelines.md

```diff
@@ -38,7 +38,7 @@ Organization of a successful MEI conference requires the active participation of
 
 ### Conference website and conference management software
 
-The OC is responsible for creating and managing a conference website where all information is promptly and clearly published, following the planned schedule. The MEI website will redirect the page [http://music-encoding.org/community/conference](http://music-encoding.org/community/conference) to the current conference website. Previous conference websites (which were managed directly by MEI until 2015) are available at [http://music-encoding.org/archive](http://music-encoding.org/archive). Since it has far-reaching effects on organizers, submitters, and registrants alike, the use of conference management software is an absolute requirement. Furthermore, unless serious legal or technical issues prevent it, the use of the ConfTool conference management system ([http://conftool.net](http://conftool.net)) is required for MEC. Potential hosts are encouraged to discuss any legal or technical difficulties with the MEI Board early in the submission process. Set-up and maintenance of the software is a responsibility that should be assigned to a member of the OC who is comfortable with the technical and time commitment. The OC member with this role provides an interface between the PC and the OC and must work well with both committees. The MEI Board will provide initial advice/assistance to the ConfTool manager and financial assistance, if funds are available.
+The OC is responsible for creating and managing a conference website where all information is promptly and clearly published, following the planned schedule. The MEI website will redirect the page [https://music-encoding.org/conference](https://music-encoding.org/conference) to the current conference website. Previous conference websites (which were managed directly by MEI until 2015) are available at [http://music-encoding.org/archive](http://music-encoding.org/archive). Since it has far-reaching effects on organizers, submitters, and registrants alike, the use of conference management software is an absolute requirement. Furthermore, unless serious legal or technical issues prevent it, the use of the ConfTool conference management system ([http://conftool.net](http://conftool.net)) is required for MEC. Potential hosts are encouraged to discuss any legal or technical difficulties with the MEI Board early in the submission process. Set-up and maintenance of the software is a responsibility that should be assigned to a member of the OC who is comfortable with the technical and time commitment. The OC member with this role provides an interface between the PC and the OC and must work well with both committees. The MEI Board will provide initial advice/assistance to the ConfTool manager and financial assistance, if funds are available.
```

Comment:
```suggestion
The OC is responsible for creating and managing a conference website where all information is promptly and clearly published, following the planned schedule. The MEI website will redirect the page [https://music-encoding.org/conference](https://music-encoding.org/conference) to the current conference website. Previous conference websites (which were managed directly by MEI until 2015) are available at [https://music-encoding.org/conference/past.html](https://music-encoding.org/conference/past.html). Since it has far-reaching effects on organizers, submitters, and registrants alike, the use of conference management software is an absolute requirement. Furthermore, unless serious legal or technical issues prevent it, the use of the ConfTool conference management system ([http://conftool.net](http://conftool.net)) is required for MEC. Potential hosts are encouraged to discuss any legal or technical difficulties with the MEI Board early in the submission process. Set-up and maintenance of the software is a responsibility that should be assigned to a member of the OC who is comfortable with the technical and time commitment. The OC member with this role provides an interface between the PC and the OC and must work well with both committees. The MEI Board will provide initial advice/assistance to the ConfTool manager and financial assistance, if funds are available.
```

Reply:
There is a dedicated page for previous conferences now. The section for conferences will probably be removed from the archive page some day.